sysdeps/lemon/generic/lemon.cpp

```diff
@@ -72,6 +72,8 @@ namespace mlibc{
 
 	void sys_exit(int status){
 		syscall(SYS_EXIT, status);
+
+		for(;;); // Prevent compiler warning
```

Comment:
This could also be done with ``__builtin_unreachable`` no?

Reply:
Yes, fixeed